### PR TITLE
Improvement/scm 960

### PR DESCRIPTION
**Pull request by mean2me**

### CODE REVIEW

File: Dropdown/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "DNN Dropdown Component",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
```
Comment (tpluscode) [inline Dropdown/package.json]: :+1:

File: Dropdown/src/Dropdown.test.js [hunk 2]
```diff
@@ -1,140 +1,140 @@
...
-        let collapsibleLabel = deepRendering.find(".collapsible-label").first();
-        collapsibleLabel.simulate("click");
-
-        const json = toJson(deepRendering);
-        expect(json).toMatchSnapshot();
-
-    });
-
-});
+jest.mock("./style.less", () => jest.fn());
```
Comment (tpluscode) [inline Dropdown/src/Dropdown.test.js]: Entire file is modified. Could you please fix?

**Comment by mean2me** [comment]
It was a github issue, since files were identical. Now it's ready to be merged.

File: Dropdown/package.json [hunk 3]
```diff
@@ -35,21 +35,27 @@
...
-    "react": "^15.3.2",
+    "react": "^15.6.2",
     "react-collapse": "~4.0.3",
-    "react-custom-scrollbars": "~4.1.2",
-    "react-dom": "^15.3.2",
+    "react-custom-scrollbars": "^4.2.1",
+    "react-dom": "^15.6.2",
     "react-motion": "^0.4.4",
     "scroll": "^2.0.0"
   }
```
Comment (mean2me) [inline Dropdown/package.json]: Some dependencies were missing and prepublishOnly script was failing. Now it's ok.

File: Dropdown/src/Dropdown.jsx [hunk 4]
```diff
@@ -228,7 +228,7 @@ class Dropdown extends Component {
             const domElement = ReactDOM.findDOMNode(optionRef);
             let offset = domElement.offsetTop;
             if (eventKey == "ArrowUp") {
-                offset = domElement.offsetBottom;
+                offset = domElement.offsetTop - domElement.clientHeight*2;
             }
             scroll.top(ReactDOM.findDOMNode(this.scrollBar).childNodes[0], offset);
         }
```
Comment (mean2me) [inline Dropdown/src/Dropdown.jsx]: It was bug I found testing locally. Now we have the right behavior when we click "arrowUp"

File: Dropdown/src/style.less [hunk 5]
```diff
@@ -116,6 +116,7 @@
                 box-sizing: border-box;
                 &.selected {
                     color: @curiousBlue;
+                    font-weight: bold;
                 }
                 &:hover {
                     background-color: @gallery;
```
Comment (mean2me) [inline Dropdown/src/style.less]: Fixed some small visual issues.

**Pull request merged**